LayoutTests/platform/ios-wk2/imported/w3c/web-platform-tests/html/semantics/document-metadata/the-meta-element/color-scheme/meta-color-scheme-attribute-changes-expected.txt

```diff
@@ -1,10 +1,10 @@
 
-FAIL Meta color-scheme initially 'dark'. assert_not_equals: CanvasText system color should be different with light and dark color schemes got disallowed value "rgb(0, 0, 0)"
-FAIL Removed name attribute from meta color-scheme. assert_not_equals: CanvasText system color should be different with light and dark color schemes got disallowed value "rgb(0, 0, 0)"
-FAIL Set meta name to color-scheme. assert_not_equals: CanvasText system color should be different with light and dark color schemes got disallowed value "rgb(0, 0, 0)"
-FAIL Set content attribute of meta color-scheme to empty string. assert_not_equals: CanvasText system color should be different with light and dark color schemes got disallowed value "rgb(0, 0, 0)"
-FAIL Set content attribute of meta color-scheme to an invalid value. assert_not_equals: CanvasText system color should be different with light and dark color schemes got disallowed value "rgb(0, 0, 0)"
-FAIL Set content attribute of meta color-scheme to 'light'. assert_not_equals: CanvasText system color should be different with light and dark color schemes got disallowed value "rgb(0, 0, 0)"
-FAIL Set content attribute of meta color-scheme to 'dark'. assert_not_equals: CanvasText system color should be different with light and dark color schemes got disallowed value "rgb(0, 0, 0)"
-FAIL Removed the content attribute of meta color-scheme. assert_not_equals: CanvasText system color should be different with light and dark color schemes got disallowed value "rgb(0, 0, 0)"
+FAIL Meta color-scheme initially 'dark'. assert_equals: Root element's color-scheme should be 'normal' expected "normal" but got "auto"
```

Comment:
The rebaselines in this folder are net progressions, the tests fail later than they used to.

The remaining failures are due to https://bugs.webkit.org/show_bug.cgi?id=224810.